platform/android/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
 ### Bug fixes
 - Load a style without an URI or JSON [#15293](https://github.com/mapbox/mapbox-gl-native/pull/15293)
+- Do not try to wake up the RunLoop if a wake is already pending [#15330](https://github.com/mapbox/mapbox-gl-native/pull/15330)
```

Comment:
Could you add - "Fixes offline downloads that could freeze after resuming".

Reply:
Added and rebased. Also updated https://github.com/mapbox/mapbox-gl-native/releases/tag/android-v8.3.0-alpha.2